components/TopBarProfileMenu.js

```diff
@@ -38,12 +41,69 @@ const HideGlobalScroll = createGlobalStyle`
 `;
 
 const UserMenuLinkEntry = props => {
+  const { profileMenuLink } = props;
   return (
-    <ListItem mb="6px">
+    <ListItem mb="6px" py={profileMenuLink ? '2' : '0'}>
       <StyledLink as={Link} fontWeight="500" fontSize="14px" lineHeight="20px" color="black.800" {...props} />
     </ListItem>
   );
 };
+UserMenuLinkEntry.propTypes = {
+  profileMenuLink: PropTypes.bool,
+};
+
+const MyAccount = props => {
```

Comment:
Maybe rename this as `UserAccount` ? To me it doesn't make sense to have `MyAccount` since this doesn't belong to a particular user. 🤔